extended/src/main/java/apoc/vectordb/ChromaDb.java

```diff
@@ -0,0 +1,220 @@
+        // to {ids: [<id1>, <id2>], vectors: [<vector1>, <vector2>]}
+        Map<Object, List> additionalBodies = listOfMapToMapOfLists(mapKeys, vectors);
+        additionalBodies.compute( "ids", (k,v) -> getStringIds(v) );
+
+        RestAPIConfig restAPIConfig = new RestAPIConfig(config, Map.of(), additionalBodies);
+        return executeRequest(restAPIConfig, urlAccessChecker)
+                .map(v -> (Map<String,Object>)v)
+                .map(MapResult::new);
+    }
+
+    @Procedure(value = "apoc.vectordb.chroma.delete", mode = Mode.SCHEMA)
+    @Description("apoc.vectordb.chroma.delete(hostOrKey, collection, ids, $configuration) - Delete the vectors with the specified `ids`")
+    public Stream<ListResult> delete(@Name("hostOrKey") String hostOrKey,
+                                     @Name("collection") String collection,
+                                     @Name("ids") List<Object> ids,
+                                     @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
+        String url = "%s/api/v1/collections/%s/delete";
+        Map<String, Object> config = getVectorDbInfo(hostOrKey, collection, configuration, url);
+
+        VectorEmbeddingConfig apiConfig = CHROMA.get().getEmbedding().fromGet(config, procedureCallContext, getStringIds(ids));
+        return executeRequest(apiConfig.getApiConfig(), urlAccessChecker)
+                .map(v -> (List) v)
+                .map(ListResult::new);
+    }
+
+    @Procedure(value = "apoc.vectordb.chroma.get", mode = Mode.SCHEMA)
+    @Description("apoc.vectordb.chroma.get(hostOrKey, collection, ids, $configuration) - Get the vectors with the specified `ids`")
+    public Stream<EmbeddingResult> query(@Name("hostOrKey") String hostOrKey,
+                                                      @Name("collection") String collection,
+                                                      @Name("ids") List<Object> ids,
+                                                      @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
+        String url = "%s/api/v1/collections/%s/get";
+        Map<String, Object> config = getVectorDbInfo(hostOrKey, collection, configuration, url);
+
+        VectorEmbeddingConfig apiConfig = CHROMA.get().getEmbedding().fromGet(config, procedureCallContext, ids);
+        return getEmbeddingResultStream(apiConfig, procedureCallContext, urlAccessChecker, tx,
+                v -> listToMap((Map) v).stream());
+    }
+
+    @Procedure(value = "apoc.vectordb.chroma.query", mode = Mode.SCHEMA)
+    @Description("apoc.vectordb.chroma.query(hostOrKey, collection, vector, filter, limit, $configuration) - Retrieve closest vectors the the defined `vector`, `limit` of results,  in the collection with the name specified in the 2nd parameter")
+    public Stream<EmbeddingResult> query(@Name("hostOrKey") String hostOrKey,
```

Comment:
sorry, that comment for the query procedure was meant for here:

I think if we should move the write behavior into a separate method, like `queryAndUpdate` or so? or `updateGraphFromQuery` ? and keep the query method read-only, otherwise read-only users can't use it and accidental write behavior will be confusing.

Reply:
Removed Mode.SCHEMA, I had accidentally left it in that initially the procedure also auto-created the vector indexes in neo4j, I removed it now.
And added procedures `queryAndUpdate` with WRITE mode